README.md

```diff
@@ -1,16 +1,22 @@
 **previously nammed secure_window**
 
+Forked from [neckaros](https://github.com/neckaros/secure_application) & [stevenspiel](https://github.com/stevenspiel/secure_application)
```

Comment:
Since it would no longer be a fork could you remove this part and merge the extended features please?